### ISSUE TITLE
accname-aam need to clarify aria-owns in Step 2F

### ISSUE THREAD

**Issue by jnurthen** [w3c/aria#538]
Need to clarify if aria-owns should be taken into account in step 2F (iii - a).

When talking about "child node"

- Chrome includes nodes owned through use of aria-owns
- FF only considers DOM children

To be honest I don't care what the result is - but I want consistency. Please clarify.

(see bug https://bugs.chromium.org/p/chromium/issues/detail?id=702811 logged against Chrome)

**Comment by Joanmarie** [comment]
And Chrome does it on purpose: https://bugs.chromium.org/p/chromium/issues/detail?id=561766. The question is why? Let's ask Dominic.

**Comment by aleventhal** [comment]
@jnurthen, I talked with both the Chrome and Firefox a11y team and the consensus is to do things the Chrome way -- that is, to include aria-owns children in the accessible name. Is there an issue with test case filed in the Mozilla bug tracker? That would be helpful

CC'ing @asurkov

**Comment by jnurthen** [comment]
This issue was moved to w3c/accname#25
